en/assembly/quick_start_pixhawk4_mini.md

```diff
@@ -0,0 +1,134 @@
+# *Pixhawk 4 Mini* Wiring Quick Start
+
+This quick start guide shows how to power the [*Pixhawk<sup>&reg;</sup> 4 Mini*](../flight_controller/pixhawk4_mini.md) flight controller and connect its most important peripherals.
+
+<img src="../../assets/flight_controller/pixhawk4mini/pixhawk4mini_iso_1.png" width="350px" title="Pixhawk4 Image" />
+
+## Wiring Chart Overview
+
+The image below shows where to connect the most important sensors and peripherals (except for motors and servos).
```

Comment:
> (except for motors and servos).

See the next comment. Image refers to this doc to explain how to connect servos and motors. I think that this text should probably be removed and the should provide overview of how to connect them.